File: ATG/app/src/main/java/com/kuhakupixel/atg/backend/NumTypeInfo.java
package com.kuhakupixel.atg.backend;

/**
 * Information about available scan type
 * its name and sizes
 */
public class NumTypeInfo {
    String name;
    Integer bitSize;

    public NumTypeInfo(String name, Integer bitSize) {
        this.name = name;
        this.bitSize = bitSize;
    }

    public String GetName() {
        return this.name;
    }

    public Integer GetBitSize() {
        return this.bitSize;
    }
}

File: ATG/app/src/main/java/com/kuhakupixel/atg/backend/NumTypeInfo.kt
package com.kuhakupixel.atg.backend

/**
 * Information about available scan type
 * its name and sizes
 */
class NumTypeInfo(var name: String, bitSize: Int) {
    var bitSize: Int

    init {
        this.bitSize = bitSize
    }

    fun GetName(): String {
        return name
    }

    fun GetBitSize(): Int {
        return bitSize
    }
}